cmd/podman/kube/play.go

```diff
@@ -247,6 +247,14 @@ func play(cmd *cobra.Command, args []string) error {
 		return errors.New("--force may be specified only with --down")
 	}
 
+	if len(playOptions.LogDriver) == 0 {
```

Comment:
I think a better check would be whether the user overrode the default or not.

Something like:
```
if !cmd.Flags().Changed("log-driver")  && playOptions.ServiceContainer {			
     playOptions.LogDriver = define.PassthroughLogging
}
```

Reply:
I didn't know this capability of the `Flags`, thanks. Fixed